src/Reflection/ReflectionParameter.php

```diff
@@ -88,89 +82,46 @@ public function __call($method, $args)
         throw new Exception\BadMethodCallException('Invalid reflection method');
     }
 
-    /**
-     * Retrieve parameter type
-     *
-     * @return string
-     */
-    public function getType()
+    public function getType(): string
     {
         return $this->type;
     }
 
-    /**
-     * Set parameter type
-     *
-     * @param string|null $type
-     * @throws Exception\InvalidArgumentException
-     * @return void
-     */
-    public function setType($type)
+    public function setType(?string $type = null): void
     {
-        if (! is_string($type) && (null !== $type)) {
-            throw new Exception\InvalidArgumentException('Invalid parameter type');
-        }
-
         $this->type = $type;
     }
 
-    /**
-     * Retrieve parameter description
-     *
-     * @return string
-     */
-    public function getDescription()
+    public function getDescription(): ?string
     {
         return $this->description;
     }
 
-    /**
-     * Set parameter description
-     *
-     * @param string|null $description
-     * @throws Exception\InvalidArgumentException
-     * @return void
-     */
-    public function setDescription($description)
+    public function setDescription(?string $description = null): void
```

Comment:
the same as on `setType` setter

Reply:
Since `$description` is optional, `null` is allowed and represents a non-present state of `$description`. I updated the constructor to also allow `null`.